spec/delivery_service_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rspec'
+require_relative '../lib/dependencies'
+
+
+RSpec.describe DeliveryService do
```

Comment:
where is test for `choose_transport_type` ?

Reply:
test added